src/main/java/org/example/assigment3.java

```diff
@@ -0,0 +1,121 @@
+package org.example;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import com.opencsv.CSVWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+public class assigment3 {
+    public static void main(String[] args) throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "/home/ehsan/Downloads/chromedriver-linux64/chromedriver");
+        WebDriver driver = new ChromeDriver();
+        String WEBURL = "https://www.shutterstock.com/";
+        driver.get(WEBURL);
+        driver.manage().window().maximize() ;
+//        search icon
+        String searchByImageXpath="//div[@class=\"MuiBox-root mui-1i4x3e6-root\"]/button";
+        driver.findElement(By.xpath(searchByImageXpath)).click();
+
+        uploadFile(driver,"/home/ehsan/Downloads/download.jpeg");
+
+        driver.findElement(By.className("mui-t7xql4-a-inherit-link")).click();
+        scrollDown(driver);
+        String photoID = getPhotoId(driver);
+        addInCsv("/home/ehsan/Downloads/PhotoID.cvs",photoID);
+        scrollUp(driver);
+        driver.findElement(By.className("mui-o90xvh-a-inherit-logo")).click();
+        Thread.sleep(10000);
+        driver.findElement(By.xpath("//div/input")).sendKeys(photoID);
+        driver.findElement(By.xpath("//div/button[@aria-label=\"Search\"]")).click();
+        Thread.sleep(10000);
+        driver.findElement(By.xpath("//div/button[@aria-label=\"Download\"]")).click();
+
+        Thread.sleep(10000);
+        driver.findElement(By.xpath("//div/button[@aria-label=\"Buy and download\"]")).click();
+        Login("","",driver);
+        addInCsv("/home/ehsan/Downloads/PhotoID.cvs",photoID);
+        takeScreenShot(driver,"/home/ehsan/Downloads/");
+    }
+
+
+
+    private static String getPhotoId(WebDriver driver) {
+        String imageID = driver.findElement(By.xpath("//div[@class=\"MuiBox-root mui-15jx5nt\"]/p")).getText();
+        String id = imageID.split("ID:")[1];
+//        System.out.println(id);
+        return id;
+    }
+
+    public static void uploadFile(WebDriver driver,String filePath) throws InterruptedException {
+        Thread.sleep(5000);
+        WebElement fileInput = driver.findElement(By.cssSelector("input[type='file']"));
+        fileInput.sendKeys(filePath);
+//        adding this because we need to wait to upload file
+        Thread.sleep(9000);
```

Comment:
use wait at some locator (pop-up) 
e.g. when you upload file pop-up displayed that your file is uploaded, use that and implement wait